bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/Epoch.scala

```diff
@@ -0,0 +1,11 @@
+package edu.gemini.spModel.core
+
+/** Epoch in Julian years. */
+final case class Epoch(value: Double)
```

Comment:
I know case classes are technically extensible but since you're really not supposed to do that I just treat them as if that weren't a possibility.  Just as `null` is technically possible but we treat it as if it doesn't exist unless working with Java.  Just wondering if you recommend `final`ing our case classes in general.